packages/ethereum-contracts/contracts/superfluid/SuperTokenFactory.sol

```diff
@@ -87,6 +91,54 @@ abstract contract SuperTokenFactoryBase is
 
     function createSuperTokenLogic(ISuperfluid host) external virtual returns (address logic);
 
+    /// @inheritdoc ISuperTokenFactory
+    function createCanonicalERC20Wrapper(ERC20WithTokenInfo _underlyingToken)
+        external
+        returns (ISuperToken)
+    {
```

Comment:
pls add a check if the list of canonical wrappers is initialized, if not revert here. Avoids a race condition with the initialization tx. Unlikely to happen, but still...